packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -1128,6 +1276,51 @@ export class Table extends TableBase {
   protected get hasIndex(): boolean {
     return this.globalSecondaryIndexes.length + this.localSecondaryIndexes.length > 0;
   }
+
+  /**
+   * Set up key properties and return the Table encryption property from the
+   * user's configuration.
+   */
+  private parseEncryption(props: TableProps): {sseSpecification: CfnTable.SSESpecificationProperty, encryptionKey?: kms.IKey} {
```

Comment:
```suggestion
  private parseEncryption(props: TableProps): { sseSpecification: CfnTable.SSESpecificationProperty, encryptionKey?: kms.IKey } {
```

But since this method is private, you could also let it's type information be inferred:

```suggestion
  private parseEncryption(props: TableProps) {
```